lib/rubocop/cop/sequel/irreversible_migration.rb

```diff
@@ -5,6 +5,8 @@ module Cop
     module Sequel
       # IrreversibleMigration looks for methods inside a `change` block that cannot be reversed.
       class IrreversibleMigration < Base
+        include RuboCop::Cop::Sequel::Helpers::Migration
```

Comment:
You may want to apply this to other migration related cops?

Reply:
Done! I've updated the PR description with these additional changes, and also created a spec helper module to wrap test code in a `Sequel.migration` block.